### ISSUE TITLE
`Failed to instantiate WASI: RuntimeError`

### ISSUE THREAD

**Issue by X7md** [wasmerio/wasmer-js#295]
I try to run `qjs` but I facing this error.

![image](https://user-images.githubusercontent.com/54203033/179220353-b283196e-d8f3-4d12-af2f-67aacd90bb99.png)

```js
Uncaught Error: Failed to instantiate WASI: RuntimeError: `
    at B2.wbg.__wbg_new_342a24ca698edd87 (Library.esm.min.js:25:11051)
    at 0024ee5a:0x7c9d3
    at 0024ee5a:0x19d5a
    at s.instantiate (Library.esm.min.js:25:6746)
    at main.mjs:20:12
```

reproduction:
https://github.com/X7md/wasmer-js-test

**Comment by ankoh** [comment]
Related: https://github.com/wasmerio/wasmer/issues/2792
The unfortunate truth is that wasmer-js currently fails to instantiate larger wasm modules on the main thread in Chrome.

**Comment by X7md** [comment]
> Related: [wasmerio/wasmer#2792](https://github.com/wasmerio/wasmer/issues/2792) Wasmer-js currently fails to instantiate larger wasm modules on the main thread in Chrome.

Yeah, I try on Worker, can't run `qjs` unfortunately...

but my `main.wasm` (Golang Hello World using TinyGo to wasi)... work fine, still don't know why.

**Comment by X7md** [comment]
> > Related: [wasmerio/wasmer#2792](https://github.com/wasmerio/wasmer/issues/2792) Wasmer-js currently fails to instantiate larger wasm modules on the main thread in Chrome.
> 
> Yeah, I try on Worker, can't run `qjs` unfortunately...
> 
> but my `main.wasm` (Golang Hello World using TinyGo to wasi)... work fine, still don't know why.

reproduction (With Worker):

https://github.com/X7md/wasmer-js-test/tree/with-worker

**Comment by X7md** [comment]
> > > Related: [wasmerio/wasmer#2792](https://github.com/wasmerio/wasmer/issues/2792) Wasmer-js currently fails to instantiate larger wasm modules on the main thread in Chrome.
> > 
> > 
> > Yeah, I try on Worker, can't run `qjs` unfortunately...
> > but my `main.wasm` (Golang Hello World using TinyGo to wasi)... work fine, still don't know why.
> 
> reproduction (With Worker):
> 
> https://github.com/X7md/wasmer-js-test/tree/with-worker

Also can't bundle it in vite build (RollUp), but work on vite dev (esBuild)

**Comment by syrusakbary** [comment]
This is weird.

We have published a new version 1.1.0... could someone verify that the issue is fixed or is still happening? Thanks!

**Comment by X7md** [comment]
for now, no more error. 

but also no output for `wasi.getStdoutString()` but I think this form my side so... I will close this as fixed.
